econplayground/templates/main/graph_confirm_delete.html

```diff
@@ -1,13 +1,26 @@
 {% extends 'base.html' %}
+{% load static econ_auth %}
 
-{% block title %}Delete Graph: {{object.title}}{% endblock %}
+{% block title %}Delete Graph: {{object.title}} | {% endblock %}
+
+{% block pagetitle %}Delete Graph{% endblock %}
 
 {% block content %}
     <p>
         Removing <strong>{{object.title}}</strong>  &mdash; Are you sure?
     </p>
     <form action="." method="post">{% csrf_token %}
-        <input class="btn btn-danger" type="submit"
-               value="Yes, delete this graph">
+        <div class="row">
+            <div class="mb-1 mr-3 pr-0">
+                <input class="btn btn-danger" type="submit"
```

Comment:
I would use a `<button>` here instead of an `<input>`, since that's what we're displaying.